fix_includes_test.py

```diff
@@ -1099,20 +1099,14 @@ class Bar;  // lines 8-8
     self.RegisterFileContents({'add_fwd_decl_with_hdr_guard': infile})
     self.ProcessAndTest(iwyu_output)
 
-  def testDoNotAddForwardDeclareInsideNamespaceWithContentfulLine(self):
-    """Tests that for 'confusing' code, we keep fwd-decl at the top."""
+  def testAddForwardDeclareInsideNamespaceWithIfDef(self):
+    """Tests that ifdef blocks are ignored when finding namespaces."""
     infile = """\
 // Copyright 2010
 
 #include "foo.h"
 
 class Bar;
-///+namespace ns {
```

Comment:
I get a little suspicious when existing tests are modified. Can you give me a brief description of what happened here?

Reply:
Gotcha, thanks!